### PR TITLE
Added the create, edit and get late policy endpoints fixes #326

### LINKED ISSUE #326: Late Policy
**What resource needs additional coverage?**
Late Policy

**What endpoints need to be covered?**
https://canvas.instructure.com/doc/api/late_policy.html

- [ ] Get a late policy
- [ ] Create a late policy
- [ ] Patch a late policy

### PR DESCRIPTION
**Pull request by kennygperez**
# Adding new endpoints

* Added get a late policy method
* Added create a late policy method
* Added patch a late policy method

Fixes #326

### CODE REVIEW

**Comment by jonespm** [comment]
This looks like it incorporates all of the API!

File: canvasapi/course.py [hunk 1]
```diff
@@ -358,6 +358,26 @@ def create_group_category(self, name, **kwargs):
...
 
+    def create_late_policy(self, **kwargs):
+        """
+        Create a late policy. If the course already has a late policy, a bad_request
+        is returned since there can only be one late policy per course.
+
+        :calls: `POST /api/v1/courses/:id/late_policy \
+        <https://canvas.instructure.com/doc/api/late_policy.html#method.late_policy.create>`_
+
+        :rtype: :class:`:rtype: :class:`canvasapi.course.LatePolicy`
```
Comment (Thetwam) [inline canvasapi/course.py]: Some weirdness with this line of the docstring. Build docs to see.

File: canvasapi/course.py [hunk 2]
```diff
@@ -554,6 +574,24 @@ def edit_front_page(self, **kwargs):
...
         return Page(self._requester, page_json)
 
+    def edit_late_policy(self, **kwargs):
+        """
+        Patch a late policy. No body is returned upon success.
+
+        :calls: `PATCH /api/v1/courses/:id/late_policy \
+        <https://canvas.instructure.com/doc/api/late_policy.html#method.late_policy.update>`_
+
+        :rtype: Boolean
```
Comment (Thetwam) [inline canvasapi/course.py]: `bool` is fine for this

File: canvasapi/course.py [hunk 3]
```diff
@@ -1410,6 +1448,23 @@ def get_groups(self, **kwargs):
...
+
+        :calls: `GET /api/v1/courses/:id/late_policy \
+        <https://canvas.instructure.com/doc/api/late_policy.html#method.late_policy.show>`_
+
+        :rtype: :class:`canvasapi.course.LatePolicy`
+        """
+
+        response = self._requester.request(
+            "GET", "courses/{}/late_policy".format(self.id)
+        )
```
Comment (Thetwam) [inline canvasapi/course.py]: Need to pass along `kwargs`

File: tests/fixtures/course.json [hunk 4]
```diff
@@ -2105,4 +2150,4 @@
 		],
 		"status_code": 200
 	}
-}
+}
```
Comment (Thetwam) [inline tests/fixtures/course.json]: please leave newline at end of file

**Pull request merged**